python/src/FunctionalChaosValidation_doc.i.in

```diff
@@ -0,0 +1,191 @@
+
+kParameter : int
+    The :math:`k` parameter of the K-Fold cross-validation method.
+    We must have :math:`1 < k \leq n` where :math:`n` is the sample size,
+    otherwise an exception is raised.
+    When :math:`k = n`, then the K-Fold method is mathematically equivalent to
+    leave-one-out, but its implementation can be slower so use leave-one-out
+    instead.
+    The default value of the :math:`k` parameter of the K-Fold
+    cross-validation method uses the `FunctionalChaosValidation-DefaultKFoldParameter`
+    key of the :class:`~openturns.ResourceMap`.
+
+See also
+--------
+FunctionalChaosAlgorithm, FunctionalChaosResult
+
+Notes
+-----
+A `FunctionalChaosValidation` object is used for the validation of a functional chaos expansion.
+It is based on the fast (analytical) leave-one-out and fast K-Fold
+cross-validation methods presented in :ref:`pce_cross_validation`.
+
+Analytical cross-validation can only be performed accurately if some
+conditions are met.
+
+- This can be done only if the coefficients of the expansion are estimated
+  using least squares regression: if the expansion is computed from integration,
+  then an exception is produced.
+- This can be done only if the coefficients of the expansion are estimated
+  using full expansion, without model selection: if the expansion is computed
+  with model selection, then an exception is produced by default.
+  This is because model selection leads to supposedly improved coefficients,
+  so that the hypotheses required to estimate the mean squared error
+  using the cross-validation method are not satisfied anymore.
+  As a consequence, using the analytical formula without taking into
+  account for the model selection leads to a biased, optimistic, mean squared
+  error.
+  More precisely, the analytical formula produces a MSE which is lower
+  than the true one on average.
+  Model selection is involved in the expansion if the
+  :class:`~openturns.LeastSquaresMetaModelSelectionFactory`
+  or :class:`~openturns.CleaningStrategy` classes are involved.
+  If the :class:`~openturns.PenalizedLeastSquaresAlgorithmFactory` class
+  is used, then no model selection is involved and the analytical
+  cross-validation methods can be used.
+  If model selection is involved, the naive methods based on the
+  :class:`~openturns.LeaveOneOutSplitter` and :class:`~openturns.KFoldSplitter`
+  classes can be used, but this can be much slower than the
+  analytical methods implemented in the :class:`~openturns.FunctionalChaosValidation`
+  class.
+  In many cases, however, the order of magnitude of the estimate from the
+  analytical formula applied to a sparse model is correct: the estimate of
+  the MSE is only slightly lower than the true value.
+  In order to enable the calculation of the analytical MSE estimator
+  on a sparse model, please set the `FunctionalChaosValidation-NoModelSelection`
```

Comment:
I propose  to use ModelSelection; its error-prone to use negation

Reply:
Very true. Fixed at 9bb11b6a7.